tests/integration_rust/install_tests.rs

```diff
@@ -853,3 +867,90 @@ async fn conda_pypi_override_correct_per_platform() {
         pep508_rs::Requirement::from_str("boltons").unwrap(),
     ));
 }
+
+#[tokio::test]
+async fn test_multiple_prefix_update() {
```

Comment:
Might be interesting to use a [`JoinSet`](https://docs.rs/tokio/latest/tokio/task/struct.JoinSet.html) and the multithreaded runtime to ensure parallel execution. As your implementation is Send in any case :)

Reply:
I cannot use joinset
it has the same limitation as tokio::spawn, that the future should be 'static
but I'm borrowing project